.github/workflows/monthly_release.yml

```diff
@@ -0,0 +1,39 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"  # every first day of the month at midnight
```

Comment:
We could set
```suggestion
on:
  workflow_dispatch
```
to trigger the workflow manually for testing purposes.

Reply:
We could also add this permanently to leave the possibility to trigger the workflow manually whenever a manual update (e.g., off the weekly period) is needed. 

This could allow us to drop the scripts weeklyUpdate.py and releasePR.py for the PR automation part - less code to maintain.